sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/version.hpp

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+/**
+ * @file
```

Comment:
What's this tag for?

Reply:
copied from https://github.com/Azure/azure-sdk-for-cpp/blob/master/sdk/core/azure-core/inc/azure/core/version.hpp